HPXMLtoOpenStudio/resources/hvac_sizing.rb

```diff
@@ -2598,6 +2599,130 @@ def self.apply_hvac_piping_load(hvac_loads, zone_loads, hvac_heating)
     zone_loads.Heat_Tot += heat_load
   end
 
+  # Iteratively calculate design SHR --> design sensible capacity --> use design sensible capacity to RECALCULATE design airflow using MJ approach
```

Comment:
Is this part of the other PR? Doesn't seem specific to GHPs?

Reply:
This is the approach for ASHP , I refactored a bit so that it's a separate method and can be called by both ghp and ashp.